src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -143,7 +181,8 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     const daysEvents = relatedVenueEvents
       .filter(
         isScheduleTimeshifted
-          ? isEventWithinDate(startOfSelectedDay)
+          ? // @debt do we need this ternary here? Can we just always use one of isEventWithinDate / isEventWithinDateAndNotFinished ?
+            isEventWithinDate(startOfSelectedDay)
           : isEventWithinDateAndNotFinished(startOfSelectedDay)
       )
```

Comment:
[debt] do we need this ternary here? Can we just always use one of `isEventWithinDate` / `isEventWithinDateAndNotFinished` ?